geps/gep-2648/index.md

```diff
@@ -102,9 +102,6 @@ simple design patterns _if they meet a set of criteria_.
 With that background and the previous example in mind, here are some rules for
 when a Policy is a Direct Attached Policy:
 
-
-* The number or scope of objects is exactly _one_ object. No label selectors or
-  lists of `targetRef` are allowed.
```

Comment:
I think this rule should stay for Direct Attached.  It is the Inherited policy that should reference multiple objects.

Reply:
Good point, I'm looking at this PR as the smallest subset of #2927 that is required for v1.1. Since both BackendTLSPolicy and BackendLBPolicy are considered "direct" policies, that seemed like the most important type of policy to extend with this capability. With that said, I think there's broad consensus as part of the discussion around #2927 that both direct and inherited will effectively be merged when the full set of changes described there are merged, so this will eventually affect all policies.

Even with that background, I think the primary differentiator between direct and inherited is if policies could affect multiple layers (ie Route and Gateway) or not. In the case of our Backend policies, they still can only affect the backend layer, so I think we're still meeting the original goals of direct policies.